doc/examples/workflow_creation.py

```diff
@@ -96,7 +96,15 @@ def run(self, input_files, text_to_append='dipy', out_dir='',
 executable file located in ``bin``.
 """
 
+"""
+
+These lines will import the required files and classes into the python file. The
+first line imports the run_flow method from the flow_runner class and the second
+line imports the AppendTextFlow method from the newly created workflow class.
```

Comment:
- I think the first sentence is not necessary (`These lines... python files.`)
- `AppendTextFlow method` -> `AppendTextFlow class`

Reply:
Hey, thanks.

I removed the first line and changed the comments to explain more clearly.